src/bentoml/_internal/frameworks/transformers.py

```diff
@@ -784,11 +814,12 @@ def import_model(
                         revision=revision,
                         force_download=force_download,
                         resume_download=resume_download,
+                        trust_remote_code=trust_remote_code,
                         **extra_hf_hub_kwargs,
                     )
                     path_to_config = transformers.utils.cached_file(
                         model_name_or_path,
-                        CONFIG_JSON_FILE_NAME,
+                        transformers.CONFIG_NAME,
```

Comment:
Do you know that this is stable?

Reply:
This is referenced by multiple files in the transformers library. I think it is unlikely they will just change this constant.